dns/py/params_test.py

```diff
@@ -73,6 +109,32 @@ def test_params(self):
         '-l,120,-Q,400,-d,/queries/abc',
         ','.join(inputs.dnsperf_cmdline))
 
+  def test_coredns_params(self):
+      values = {
+          'coredns_cpu': 100,
+          'coredns_cache': 200,
+      }
+
+      inputs = Inputs(make_mock_coredns_deployment_yaml(),
+          make_mock_coredns_configmap_yaml(), [])
+
+      for param in PARAMETERS:
+        if param.name not in values: continue
```

Comment:
You should avoid multiple statements on one line if possible.
http://pep8.readthedocs.io/en/stable/intro.html?highlight=E701
http://pep8.readthedocs.io/en/stable/_modules/pycodestyle.html?highlight=E701